aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsFluentClientDecorator.kt

```diff
@@ -179,7 +179,24 @@ private class AwsFluentClientExtensions(types: Types) {
                 /// Creates a new client from a shared config.
                 ##[cfg(any(feature = "rustls", feature = "native-tls"))]
                 pub fn new(sdk_config: &#{aws_types}::sdk_config::SdkConfig) -> Self {
-                    Self::from_conf(sdk_config.into())
+                    let conf: crate::Config = sdk_config.into();
```

Comment:
The `from_conf` function (below this one) also needs to be updated. Currently it's ignoring the config's connector.

The code added to `new` can probably just be moved into `from_conf`.